package-lock.json

```diff
@@ -0,0 +1,12427 @@
+{
```

Comment:
I think this file shouldn't be here. YOu already have a `yarn.lock` and form the CI configuration it looks like a Yarn project and not an NPM project.

Reply:
I heard that npm has become better than yarn in recent releases and that many people are preferring npm now...